### ISSUE TITLE
angionet output

### ISSUE THREAD

**Issue by qazalkz** [kritiyer/AngioNet#1]
hello. could you please tell me what is angionet output?
I've got 512, 512, 2 !

**Comment by kritiyer** [comment]
Hello, the output size is correct. The image size is 512x512 and there are two channels for the two classes (background = 0 and vessel = 1). You can get the binary segmentation image using softmax, or since there are only two classes you can simply threshold one of the channels.

**Comment by qazalkz** [comment]
thank you

**Comment by DHAiRYA2048** [comment]
without fine-tuning, i am getting the output as show below. is it expected or am i doing something wrong?
![image](https://github.com/kritiyer/AngioNet/assets/66826063/bece0729-5342-412d-8521-ea228872736f)

**Comment by kritiyer** [comment]
Yes, this is expected as we have not provided pretrained weights for the
whole model. You will need to fine-tune the model after loading one of the
pretrained weights for the Deeplab backbone. We originally had plans to
provide a platform where the pretrained weights could be accessed for
inference but it was never implemented.

**Comment by DHAiRYA2048** [comment]
Got it, please correct me if I'm wrong.

   1. use the weights from the TF website in this line (deeplab_model =
   Deeplabv3(weights=DL_weights, backbone="xception", input_shape =
   (512,512,3), classes=2))
   2. load the angionet (angionet_model.py) architecture
   3. fine tune with my data

Dhairya Shah

"Excellence is never an accident"

**Comment by kritiyer** [comment]
Yes, that's correct. Good luck!

**Comment by DHAiRYA2048** [comment]
Awesome, thank you so much!
